### ISSUE TITLE
Can't batch edit rights

### ISSUE THREAD

**Issue by hannahlwang** [projecthydra/sufia#3051]
### Descriptive summary

Hi, I'm at UNC with @julesies. We have Sufia 7.2. In the Description tab of the Batch Edit view with multiple works selected, I try to change the Rights by selecting a value in the drop-down. I get a "Changes Saved" message, but when I view each object, the values in the Rights fields are unchanged. This is only occurring with the Rights field -- batch editing for other metadata fields seems to work as expected.

### Expected behavior

When I batch edit the Rights for multiple objects, I expect the values in the Rights fields for all the selected objects to change to that new value.

### Actual behavior

When I batch edit the Rights for multiple objects, the values in the Rights fields are unchanged.

### Steps to reproduce the behavior

1. As a logged in user, go to "My Works"
2. Select multiple works and click "Edit Selected"
3. In the Descriptions tab, click "Rights" and select a value in the drop-down menu
4. Click "Save Changes"
5. View all selected works and look at the Rights field in description

**Comment by hannahlwang** [comment]
@jcoyne Is batch editing in the Descriptions tab also addressed in that issue? I was looking at the Rights field in the Descriptions tab, rather than the Permissions tab. In any event, we will hopefully test 7.3 soon and let you know

**Comment by jcoyne** [comment]
@hannahlwang my mistake, I was confusing "rights" with "permissions"

**Comment by mjgiarlo** [comment]
@hannahlwang @jcoyne I could not replicate this in the latest Hyrax build. I succeeded at both changing the Rights field for three works, and at adding a new Rights field, with two separate attempts.

@hannahlwang Were you using Chrome? Or could what you're seeing here have been related to a cross-browser behavior w/r/t the checkboxes being cached in Firefox/Safari (the other bug you entered)? It's also possible we've made fixes since Sufia 7.2.

I'm inclined to close this issue, but let me know if I should keep it open.
